pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -260,27 +260,14 @@ func (r *ReconcileKnativeServing) checkDeployments(instance *servingv1alpha1.Kna
 // Check for all dependencies
 func (r *ReconcileKnativeServing) checkDependencies(instance *servingv1alpha1.KnativeServing) error {
 	defer r.updateStatus(instance)
-	checkError := func(err error, instance *servingv1alpha1.KnativeServing) error {
-		msg := fmt.Sprintf("Istio not detected; please install ServiceMesh")
-		instance.Status.MarkDependencyMissing(msg)
-		log.Error(err, msg)
-		return err
-	}
-	smmr := &maistrav1.ServiceMeshMemberRoll{}
-	if err := r.client.Get(context.TODO(), client.ObjectKey{Namespace: "istio-system", Name: "default"}, smmr); err != nil {
-		return checkError(err, instance)
-	}
-	found := func(smmr *maistrav1.ServiceMeshMemberRoll) bool {
-		for _, member := range smmr.Status.ConfiguredMembers {
-			if member == "knative-serving" {
-				return true
-			}
-		}
-		return false
-	}
-	if !found(smmr) {
-		return checkError(errors.New("knative-serving namespace is not a configured member in serviceMeshMemberRoll"), instance)
+
+	istio := schema.GroupVersionKind{Group: "networking.istio.io", Version: "v1alpha3", Kind: "virtualservice1"}
```

Comment:
I think this is a typo, but I'd rather check for "gateway" kind anyway, because that's the type our manifest depends on. (I know I had "virtualservice" in there before, but I think that was a mistake)